README.md

```diff
@@ -843,6 +843,76 @@ This allows you to:
 - Have multiple Odoo instances in each node.
 - Add an SSL layer automatically and for free.
 
+###### Backups
+
+Out-of-box Doodba has following schedule:
+
+```
+# Makes postgres dump for all databases. Uploads dumps and current filestore
+JOB_200_WHEN=daily weekly
+
+# Incremental backup of all available files
+JOB_300_WHEN=daily
+
+# Full backup of all available files including postgres dumps if they were made before
+JOB_500_WHEN=weekly
+```
+
+Available options to schedule tasks:
+
+- 15min
+- hourly
+- daily
+- weekly
+- monthly
+- never
+
+[Default values for the periodicities](https://github.com/Tecnativa/docker-duplicity/blob/master/Dockerfile):
+
+```
+CRONTAB_15MIN=*/15 * * * *
+CRONTAB_HOURLY=0 * * * *
+CRONTAB_DAILY=0 2 * * MON-SAT
+CRONTAB_WEEKLY=0 1 * * SUN
+CRONTAB_MONTHLY=0 5 1 * *
+```
+
+You can redefine periodicity meanings in `.env` file.
+
+To make backup immediatly execute following command:
+
+```sh
+# make postgres dump
+docker-compose exec backup sh -c 'echo $JOB_200_WHAT | sh'
```

Comment:
```suggestion
docker-compose exec backup /etc/periodic/daily/jobrunner
```

Reply:
It won't work if a user decides to change schedule for JOB_200_WHEN, e.g. 

    JOB_200_WHEN=weekly

or

    JOB_200_WHEN=hourly

Reply:
Maybe a note about that? This method is better IMHO because makes a full backup process in a single command...

Reply:
What about 

```
docker-compose exec backup sh -c 'echo $JOB_200_WHAT | sh && dup full $SRC $DST'
```

It runs in a single command and doesn't need workaround when the schedule is changed

Reply:
It's the same, if you added a JOB_201, it won't run, so the backup process won't be full. :man_shrugging: 

I think it's better to document the default behavior in a more general fashion and document the details in https://github.com/Tecnativa/docker-duplicity directly.

Reply:
> It's the same, if you added a JOB_201, it won't run, so the backup process won't be full. 

It seems full for me in doodba context. But it makes sense in general context. I'd prefer to keep my version here and add yours to the docker-duplicity

Reply:
Hmm I'm thinking on another approach. How about this? :thinking: 

```sh
# If your backups are configured daily, this will execute a full backup
backup_periodicity=daily # Change to execute other periodicities jobs
docker-compose exec backup /etc/periodic/$backup_periodicity/jobrunner
```

The point of doodba is making things easy, so I still think the jobrunner approach is simpler, easier to read and makes what you expect. It even sends you a notification email!

Well, you're the newcomer here, so what do you think would make you faster to get it up and running?

Reply:
As a newcomer, I'd not change the default backup schedule :smiley: . I'll come with new version of the doc soon

Reply:
Done